syntax = "proto3";

package airflower_protobufs;

service VLRStrategia {
    rpc GetVLRThreshold(VLRThresholdRequest) returns (VLRThresholdResponse) {}
}

message VLRThresholdRequest {
    string strategy = 1;
}

message VLRThresholdResponse {
    int64 threshold = 1;
}
